### PR TITLE
Disable multiple inputs into same simulate box

### LINKED ISSUE #1898: [TASK]: Disable multiple configs going into the same simulation node
Only allow a single config for a simulation node.
If user wants to simulate another config, they need to do that via an additional simulation node.

### PR DESCRIPTION
**Pull request by Jami159**
Resolves #1898 #1899

### CODE REVIEW

**Comment by mwdchang** [comment]
There are a decent number of list-processing logic in the simulate boxes. 

If indeed we don't need to accept multiple configurations anymore we might as well try to simplify them, as I recall they were rather quickly hacked together, so cleaning things up would make the code more manageable.

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/simulationservice/SimulationCiemssServiceProxy.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import software.uncharted.terarium.hmiserver.models.simulationservice.JobResponse;
 
 
-@FeignClient(name = "ciemss-service", url = "${terarium.dataservice.url}", path = "/ciemss-service")
+@FeignClient(name = "ciemss-service", url = "${ciemss-service.url}", path = "")
```
Comment (dvince2) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/simulationservice/SimulationCiemssServiceProxy.java:13]: I believe here you can fully drop `path`

Comment (Jami159) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/simulationservice/SimulationCiemssServiceProxy.java]: done!

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/simulationservice/SimulationServiceProxy.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import software.uncharted.terarium.hmiserver.models.simulationservice.JobResponse;
 
-@FeignClient(name = "simulation-service", url = "${terarium.dataservice.url}", path = "/simulation-service")
+@FeignClient(name = "simulation-service", url = "${simulation-service.url}", path = "")
```
[the remark this answers is not in the document]
Comment (Jami159) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/simulationservice/SimulationServiceProxy.java]: done!

File: packages/client/hmi-client/src/workflow/ops/simulate-julia/tera-simulate-node-julia.vue [hunk 3]
```diff
@@ -161,33 +191,42 @@ watch(
...
-});
+	const configId = props.node.inputs[0].value?.[0];
+	if (configId) {
+		const modelConfig = await getModelConfigurationById(configId);
+		const parameters = modelConfig.configuration.semantics.ode.parameters;
+		csvData.forEach((row) =>
+			parameters.forEach((parameter) => {
+				row[parameter.id] = parameter.value;
+			})
+		);
```
Comment (mwdchang) [inline packages/client/hmi-client/src/workflow/ops/simulate-julia/tera-simulate-node-julia.vue:208]: What makes you assume that `csvData` and `ode.parameters` will be in the same order?

**Review by mwdchang — CHANGES_REQUESTED** [review]
There are a few confusing points that should be addressed
- You removed the `v-for" context for the tera-simulate-chart, but you still have a `:key` assignment, so what exactly do you want to do?
- I think you might be mangling the `:run-results` assignment in tera-simulate-chart, can you clarify what is been passed in? Seems like you are passing in the value but the chart is expecting an object-map?

**Comment by mwdchang** [comment]
How does "add-chart" work under this new logic?

**Review by mwdchang — APPROVED** [review]

File: packages/client/hmi-client/src/workflow/ops/simulate-julia/tera-simulate-node-julia.vue (hunk 3 above)
Comment (Jami159) [reply packages/client/hmi-client/src/workflow/ops/simulate-julia/tera-simulate-node-julia.vue:209]: marking as resolved for now as we'll take a deeper look at this in the future as per our conversation earlier

**Pull request merged**